client/trino-client/src/main/java/io/trino/client/StageStats.java

```diff
@@ -40,6 +40,8 @@
     private final long processedRows;
     private final long processedBytes;
     private final long physicalInputBytes;
+    private final long physicalInputDataSize;
+    private final long inputDataSize;
```

Comment:
it's unclear what is difference between `physicalInputDataSize` and `inputDataSize`

Reply:
Agreed. But it's pre-existing within the entire engine.

IIRC physical = data read from filesystem, input = data (decoded blocks) processed by engine.

Reply:
> input = data (decoded blocks) processed by engine.

What's the difference between `inputDataSize` and `processedBytes`? btw: in engine we call it `io.trino.execution.QueryStats#processedInputDataSize`

> Agreed. But it's pre-existing within the entire engine.

Sure. That is the case. However, we might be extra careful how we expose stats to client. And since we start basically from scratch, maybe we should do more generic so that it works better going forward.